Don't allow PAM module to get unloaded

It uses libraries which can't handle being unloaded very
gracefully.

http://bugs.freedesktop.org/show_bug.cgi?id=31503

diff -up fprintd-0.2.0/pam/Makefile.am.dont-ever-unload fprintd-0.2.0/pam/Makefile.am
--- fprintd-0.2.0/pam/Makefile.am.dont-ever-unload	2010-11-09 10:55:30.452135193 -0500
+++ fprintd-0.2.0/pam/Makefile.am	2010-11-09 10:55:32.954860614 -0500
@@ -5,7 +5,7 @@ pammoddir=$(libdir)/security
 
 pam_fprintd_la_SOURCES = pam_fprintd.c $(MARSHALFILES)
 pam_fprintd_la_CFLAGS = -fPIC $(WARN_CFLAGS) $(GLIB_CFLAGS)
-pam_fprintd_la_LDFLAGS = -avoid-version -module
+pam_fprintd_la_LDFLAGS = -avoid-version -module -Wl,-z,nodelete
 pam_fprintd_la_LIBADD = $(PAM_LIBS) $(GLIB_LIBS)
 
 MARSHALFILES = marshal.c marshal.h
